README.md

```diff
@@ -1,2 +1,115 @@
-# KokkosP Profiling Tools
-Performance profiling tools that can be optionally integrated into Kokkos-based applications and libraries.
+
+
+# Kokkos Tools
+
+Kokkos Tools provide a light-weight set of profiling and debugging utilities, which interface with instrumentation hooks built directly into the Kokkos runtime. Compared to 3rd party tools these tools can provide much cleaner, context specific information: in particular, they allow kernel centric analysis and they use labels provided to Kokkos objects (kernel launches and views).
+
+Under most circumstances the profiling hooks are compiled into Kokkos executables by default. That means the tools work for your existing Kokkos applications, assuming that the profiling hooks version is compatible with the tools version. No recompilation or changes to your build procedures are required.
+
+Note: `Kokkos_ENABLE_LIBDL` must be on to load profiling hooks dynamically. It should be on by default however.
```

Comment:
Thanks, 'configured' makes sense. We don't need to expand on this since the average user of Kokkos tools and Kokkos oughtn't need to mess with the configuration.